tfhe-zk-pok/src/proofs/pke.rs

```diff
@@ -232,18 +261,24 @@ where
             c_hat,
             c_y,
             pi,
-            c_hat_t,
-            c_h,
-            pi_kzg,
+            compute_load_proof_fields: optional_fields,
```

Comment:
`optional_fields` could be renamed `compute_load_proof_fields`
Applies to other places

Reply:
done !